lib/backend/k8s/k8s_test.go

```diff
@@ -1284,7 +1284,12 @@ var _ = Describe("Test Syncer API for Kubernetes backend", func() {
 		})
 
 		By("Deleting the Pod and expecting the wep to be deleted", func() {
-			err = c.ClientSet.CoreV1().Pods("default").Delete(pod.ObjectMeta.Name, &metav1.DeleteOptions{})
+			var zero int64
+			policy := metav1.DeletePropagationBackground
```

Comment:
Interesting - what are the dependents that we need to propagate deletion to now?

Reply:
I'm afraid I don't know in detail; but I could observe in the logs that libcalico-go was never seeing a Deleted event for the Pod, and then it was a bit of trial and error and luck that discovered this 'solution'.

That said, I guess it could be the volume that is automatically set up to hold the default service account?  As in the kube-controller PR that was one of the things that showed up as different between `pod` and `podNow`.

Would you like me to dig more on this before we finalize these PRs?  Or are you happy for us to run with this and keep our ears open for more detail in k8s work overall?

Reply:
Nah, I'm happy to merge as is. I was just curious.

Reply:
OK, thanks.  (Also I meant to say "the volume that is automatically set up to hold the default service account *secret*", above.)